subjects/cybersecurity/image-inspector/audit/README.md

```diff
@@ -0,0 +1,96 @@
+#### General
+
+##### Check the Repo Content
+
+Files that must be inside the repository:
+
+- Detailed documentation in the `README.md` file.
+- Source code for the Image Inspector tool.
+- Any required configuration files and scripts for running the tool.
+
+###### Are all the required files present?
+
+##### Play the Role of a Stakeholder
+
+Organize a simulated scenario where the student takes on the role of a Digital Forensics Expert and explains their solution and knowledge to a team or stakeholder. Evaluate their grasp of the concepts and technologies used in the project, their communication efficacy, and their critical thinking about their solution and the knowledge behind this project.
+
+Suggested role play questions include:
+
+- What is metadata in the context of digital images, and why is it important?
+- How does steganography work, and what are its potential uses and risks?
+- What challenges did you face while developing the Image Inspector tool, and how did you address them?
+- How does the Image Inspector tool help in identifying hidden information within images?
+- How can this tool be used in real-life digital forensics or cybersecurity scenarios?
+- What ethical considerations should be taken into account when analyzing images for hidden data?
+
+###### Were the students able to answer all the questions?
+
+###### Did the students demonstrate a thorough understanding of the concepts and technologies used in the project?
+
+###### Were the students able to communicate effectively and justify their decisions and explain the knowledge behind this project?
+
+###### Were the students able to evaluate the value of this project in real-life scenarios?
+
+###### Did the students demonstrate an understanding of ethical and legal considerations related to digital forensics?
```

Comment:
Same as the above question i see that is duplicated question

Reply:
this question for the auditor and the other for the student

Reply:
same as above

Reply:
Done in this PR: https://github.com/01-edu/public/pull/2731